pilot/pkg/ambient/controller/waypoint_controller.go

```diff
@@ -80,21 +83,11 @@ func NewWaypointProxyController(client kubelib.Client, clusterID cluster.ID, con
 		client:       client,
 		cluster:      clusterID,
 		injectConfig: config,
-		patcher: func(gvr schema.GroupVersionResource, name string, namespace string, data []byte, subresources ...string) error {
-			c := client.Dynamic().Resource(gvr).Namespace(namespace)
-			t := true
-			_, err := c.Patch(context.Background(), name, types.ApplyPatchType, data, metav1.PatchOptions{
-				Force:        &t,
-				FieldManager: "waypoint proxy controller",
-			}, subresources...)
-			return err
-		},
 	}
 
 	rc.queue = controllers.NewQueue("waypoint proxy",
 		controllers.WithReconciler(rc.Reconcile),
-		controllers.WithMaxAttempts(5))
-
+		controllers.WithMaxAttempts(maxReconcileAttempts))
```

Comment:
Nit: IMO (but probably not others opinion) this refactor of moving the constant just masks readability, not improves it